### ISSUE TITLE
Ignore files nested 2 levels deep

### ISSUE THREAD

**Issue by georgiee** [pattern-lab/patternlab-node#633]
Hello Team,
I put folders with assets in each of my pattern subtype folders. Those assets are js files, partials or whatever you want to put in there. 

File structure is like this:
```
//works fine
pattern/subtype/pattern-file.mustache
pattern/subtype/pattern-file.js

//these will trigger the 2 level warning
pattern/subtype/sources
pattern/subtype/sources/some-file.js
pattern/subtype/sources/_some-pattern.mustache
```

But I get a warning for those nested folders:

> "A pattern file [...] was found greater than 2 levels deep from [...]"
> It's strongly suggested to not deviate from the following structure under _patterns/
> While Pattern Lab may still function, assets may 404 and frontend links may break. Consider yourself warned.

The warning suggests that patternlab at least tries to process those files. I have concerns, that my files have side effects on the generating part of patternlab. You state yourself that it's strongly not suggested. So can we please just ignore files if too deep? 

This would save us from that huge list of warnings and ensure that no file will ever take part in the pattern processing.

What do you think? Thanks!

**Comment by david-moldenhauer** [comment]
Hi @georgiee,

I've forked patternlab-node and added a small *fix* to fit my needs. Similar to you I wanted the following file structure:
- 01-molecules/
  - 00-blocks/
    - 00-media-block/
      - _media-block.scss (only variables, mixins...)
      - media-block.md
      - media-block.hbs
      - media-block-include.scss
      - media-block.json
      - media-block~variant2.json
this will work without breaking the navigation, the media-block is handled as 01-molecules-00-media-block. You can find the forked version here:  https://github.com/david-moldenhauer/patternlab-node

cheers

**Comment by Hydraner** [comment]
I know this is an old issue, but I recently added this ability to my patternlab by exposing a new config, see PR

**Comment by JosefBredereck** [comment]
Fixed with #1143 in [V5.12.0](https://www.npmjs.com/package/@pattern-lab/core)
